js/jsonld.js

```diff
@@ -1646,6 +1646,30 @@ jsonld.cache = {
 };
 
 /**
+ * Accept header.
+ */
+jsonld.acceptHeader = 'application/ld+json, application/json';
```

Comment:
This isn't a default, I just reproduced the behaviour previously implemented on the node loader. 

But it in my opinion, it should only be default. The current implementation is far too restrictive on this `accept` header. The `RangeError` exception should only be raised when the `accept` header doesn't contain `application/ld+json` nor `application/json`.

This way it could be overridden for more exotic things like `application/ld+json, text/html, application/json`, that would still be valid.

What do you think?